File: IKeep.Web/ClientApp/src/services/dtos/MultimediaDto.js
class MultimediaDto
{
    constructor(json)
    {
        if(json)
        {
            this.MultimediaId = json.multimediaDto;
            this.ParentId = json.parentId;
            this.Base64 = json.base64;
            this.CreateDate = json.createDate;
            this.Name = json.name;
            this.Description = json.description;
            this.Downloads = json.downloads;
        }
        else
        {
            this.MultimediaId = "00000000-0000-0000-0000-000000000000";
            this.ParentId = "";
            this.Base64 = "";
            this.CreateDate = "";
            this.Name = "";
            this.Description = "";
            this.Downloads = "";
        }
    }
}

export default MultimediaDto;

File: IKeep.Lib/Services/Dtos/MultimediaDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IKeep.Lib.Services.Dtos
{
    public class MultimediaDto
    {
        public Guid MultimediaId { get; set; }
        public Guid ParentId { get; set; }
        public string Base64 { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string Name { get; set; }
        //public string Extension { get; set; }
        public int? Downloads { get; set; }
        public string Description { get; set; }
        
    }
}
